core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -652,6 +699,20 @@ message ShuffleReadMetricDistributions {
   repeated double remote_bytes_read = 6;
   repeated double remote_bytes_read_to_disk = 7;
   repeated double total_blocks_fetched = 8;
+  repeated double remote_reqs_duration = 9;
+  ShufflePushReadMetricDistributions shuffle_push_read_metrics_dist = 10;
+}
+
+message ShufflePushReadMetricDistributions {
+  repeated double corrupt_merged_block_chunks = 1;
+  repeated double merged_fetch_fallback_count = 2;
+  repeated double remote_merged_blocks_fetched = 3;
+  repeated double local_merged_blocks_fetched = 4;
+  repeated double remote_merged_chunks_fetched = 5;
+  repeated double local_merged_chunks_fetched = 6;
+  repeated double remote_merged_bytes_read = 7;
+  repeated double local_merged_bytes_read = 8;
+  repeated double remote_merged_reqs_duration = 9;
 }
```

Comment:
+CC @gengliangwang 
Can you please take a look specifically at the protobuf definitions and make sure they are aligned with your mental model of how this should be modeled ?

Thanks !

Reply:
Yes, the protobuf definitions LGTM

Reply:
Thanks @gengliangwang !